### PR TITLE
1562004 on 3.9

### PR DESCRIPTION
**Pull request by jcantrill**
3.9 backport of #1070

### CODE REVIEW

File: fluentd/README.md [hunk 1]
```diff
@@ -20,3 +20,18 @@ is already created by ansible installer.
...
+
+## Utilities
+### sanitize_msg_chunks
+Sanitize file buffer chunks by removing corrupt records.
+
+There are known [cases](https://bugzilla.redhat.com/show_bug.cgi?id=1562004) where fluentd is stuck processing
+messages that were buffered to corrupt file buffer chunks. This utility is run manually and deserializes each
+file chunk, perform a limited set of operations to confirm message validity. Use this utility by:
+
+* Stopping fluentd
```
Comment (ewolinetz) [inline fluentd/README.md]: When you say to stop fluentd you mean rsh into the pod and then kill the running process? Doesn't the pod go away when we do this?

Comment (richm) [reply fluentd/README.md]: to stop fluentd on a given node - use `oc label node $node logging-infra-fluentd-`, then use `oc get pods` or `ps` to make sure fluentd is not running.

Comment (ewolinetz) [reply fluentd/README.md]: So we aren't running this utility from within the fluentd pod?

**Comment by jcantrill** [comment]
/retest

@richm Maybe we no longer need this util if we are properly handling in the elastic plugin via the DLQ

**Comment by richm** [comment]
@jcantrill yeah, good point

**Comment by jcantrill** [comment]
Closing as changes to the Elasticsearch fluentd plugin will handle those issues now.

**Pull request closed without merging**